shell/platform/tizen/channels/platform_channel_stub.cc

```diff
@@ -7,24 +7,8 @@
 namespace flutter {
 
 PlatformChannel::PlatformChannel(BinaryMessenger* messenger,
-                                 TizenRenderer* renderer) {
-  renderer_ = nullptr;
-}
+                                 TizenRenderer* renderer) {}
 
 PlatformChannel::~PlatformChannel() {}
 
-void PlatformChannel::HandleMethodCall(
-    const MethodCall<rapidjson::Document>& call,
-    std::unique_ptr<MethodResult<rapidjson::Document>> result) {}
-
-namespace clipboard {
-
-void GetData(const MethodCall<rapidjson::Document>& call,
-             std::unique_ptr<MethodResult<rapidjson::Document>> result) {}
-
-void SetData(const MethodCall<rapidjson::Document>& call,
-             std::unique_ptr<MethodResult<rapidjson::Document>> result) {}
```

Comment:
Does excluding these functions from stub mean that there are no plans to test any more?

Reply:
Well, I think we can move the namespace `clipboard` into a separate header file if we'd like to test the methods later.

Reply:
Currently,I am  trying to make a unit test of a platform channel.
For this purpose, I have a plan to change these functions to a protected method of the platform channel.
What do you think of it?

Reply:
I'm okay with it although I'm not the original author of the functions. Actually the current platform_channel implementation is quite messy and I had a plan to refactor it before we write unit tests for it.

Reply:
Oh if you already have plans I'll do something else.